### PR TITLE
322 update style & syntax for rubocop

### PR DESCRIPTION
**Pull request by b264**
I got one file fixed up.  All 1852 offences will take some time...this fixes 96 of them

https://github.com/ctran/annotate_models/issues/322

### CODE REVIEW

File: bin/annotate [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
-  abort "Please run annotate from the root of the project."
+unless File.exist?('./Rakefile') || File.exist?('./Gemfile')
+  abort 'Please run annotate from the root of the project.'
 end
 
 require 'rubygems'
 begin
   require 'bundler'
   Bundler.setup
-rescue Exception => e
```
Comment (b264) [inline bin/annotate]: Rubocop will not allow this.  This could have side effects.

**Comment by b264** [comment]
This lowers it to 1681.  Some of these will require massive refactoring/rewriting.

**Comment by ctran** [comment]
Thanks for the help!

**Comment by ctran** [comment]
This is not needed anymore?

**Comment by b264** [comment]
I think except for that rescue line it could help.  I am not comfortable removing that, not _fully_ understanding why it was put there in the first place.  I will remove that

**Comment by b264** [comment]
closed as superseded by https://github.com/ctran/annotate_models/pull/328

**Pull request closed without merging**